app/pages/api/allteamstats.js

```diff
@@ -2,7 +2,8 @@ import { getAllTeamStats } from "../../apis/stats.js";
 
 export default async function allteamstats(req, res) {
     try {
-        const data = await getAllTeamStats();
+        const page = (req.query.page ?? 1) - 1;
```

Comment:
Be careful, if you pass a negative number to OFFSET, it will error the database, preferably prevent that by checking ourselves

Reply:
true but they're prob going to be buttons on the page to navigate so we control the offset

Reply:
Even so, users are able to access this api with whatever values they want, so we should try to minimize errors